db/featured.ts

```diff
@@ -69,17 +58,6 @@ export const FEATURED_PROJECTS = [
     twitterURL: 'https://twitter.com/kubewarden',
     documentationURL: '',
   },
-  {
-    color: '#004D93',
-    logo: 'epinio.svg',
-    tag: 'infrastructure',
-    name: 'Epinio',
-    description: 'An opinionated platform that runs on Kubernetes and takes you from code to URL in one step.',
-    href: 'https://epinio.io/',
-    repositoryURL: 'https://github.com/epinio',
-    twitterURL: '',
-    documentationURL: '',
-  },
```

Comment:
I would hold off on Epinio, as well.

Reply:
it's not a suse project though anymore? so I think the change is ok.